modules/light-clients/08-wasm/testing/wasm_endpoint.go

```diff
@@ -23,6 +23,9 @@ var (
 	contractClientState    = []byte{1}
 	contractConsensusState = []byte{2}
 	ErrMockContract        = errors.New("mock contract error")
+	MockClientStateBz      = []byte("client-state-data")
+	MockValidProofBytes    = []byte("valid proof")
```

Comment:
there are also some client state/proof byte mocks in `upgrade_test.go` which can be replaced by these

Reply:
Ahh, yes I noticed them too, didn't have too many occurrences so I ignored them, but ya I'll make the changes and push them right now